specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -377,7 +377,12 @@ Unit and integration tests will be introduced by following the official Airflow
 
 The communication between VDK Provider and Control Service will be secured through the
 use of HTTPS and OAuth 2.0. The most commonly used OAuth 2.0 flows will be supported (
-e.g. Client Credentials Flow, Refresh Token Flow, etc.).
+e.g. Client Credentials Flow, Refresh Token Flow, etc.) by using `vdk-control-cli`'s authentication
+functionality. This would make `vdk-control-cli` a hard dependency of the Provider, though
+it is possible for the functionality to be moved as a separate library.
+
+Currently, `vdk-control-cli` fully supports the Refresh Token Flow, and provides partial support for
```

Comment:
vdk-control-cli provides support for [Authorization Code Flow with Proof Key for Code Exchange (PKCE)](https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow-with-proof-key-for-code-exchange-pkce).


What do you mean by partial support for the Client Credentials Flow ?


Also What is refresh token flow ? I assume you mean either refresh token used as part of [Autorization Code flow](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1) or the custom API token flow we have .. ? 

Please include links to new terms you use (in a similar way as I did above) or define them in the Terms section if they are our own terminology. Otherwise, there's more room for ambiguity and miscommunication.

Reply:
Reworded. I don't think we need new terms when it comes to Oauth2.

Reply:
Please include a link to the specification - so it's clear what you mean. 
In case of Refresh Token flow link to https://datatracker.ietf.org/doc/html/rfc6749#section-1.5 (I assume)

Reply:
But it is a core configuration, right?

Reply:
For example, Databrick provider uses password filed for client_sercret.

Reply:
I see no problem in reusing the password field, but I am ok with both approaches.

Reply:
Added links.